[Iron] Build failure on buildfarm for `webots_ros2_driver`

[Full log](https://build.ros2.org/view/Ibin_uJ64/job/Ibin_uJ64__webots_ros2_driver__ubuntu_jammy_amd64__binary/10/)

```
14:34:06 /tmp/binarydeb/ros-iron-webots-ros2-driver-2023.1.1/src/WebotsNode.cpp:17:10: fatal error: yaml-cpp/yaml.h: No such file or directory
14:34:07    17 | #include <yaml-cpp/yaml.h>
14:34:07       |          ^~~~~~~~~~~~~~~~~
14:34:07 compilation terminated.
```

The package.xml is missing a `<depend>libyaml-cpp-dev</depend>` along with a necessary includes + links in CMakeLists.txt

--- a/scripts/ci_after_init.bash
+++ b/scripts/ci_after_init.bash
@@ -14,7 +14,7 @@ NIGHTLY_DATE=`date --date="${LAST_NIGHTLY_DAY_OLD} day ago" +"%-d_%-m_%Y"`
 WEBOTS_NIGHTLY_VERSION="nightly_${NIGHTLY_DATE}"
 
 apt update
-apt install -y wget dialog apt-utils psmisc lsb-release git libyaml-cpp-dev
+apt install -y wget dialog apt-utils psmisc lsb-release git
 wget https://github.com/cyberbotics/webots/releases/download/${WEBOTS_NIGHTLY_VERSION}/webots_${WEBOTS_RELEASE_VERSION}_amd64.deb -O /tmp/webots.deb
 apt install -y /tmp/webots.deb xvfb
 

--- a/webots_ros2_driver/package.xml
+++ b/webots_ros2_driver/package.xml
@@ -27,6 +27,7 @@
   <depend>tinyxml2_vendor</depend>
   <depend>webots_ros2_msgs</depend>
   <depend>webots_ros2_importer</depend>
+  <depend>yaml-cpp</depend>
 
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>

